samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -267,20 +273,41 @@ public TestRunner addOutputStream(CollectionStream stream) {
     return this;
   }
 
+
```

Comment:
Commenting here since github doesn't allow me to comment on the top(Line 86). 

Currently we use `PassThroughCoordinator` for coordination and `SingleContainerGrouper` as `TaskNameGrouper`. This means that `TestRunner` api  supports single containers alone. So it will be better to remove `MULTI_CONTAINER` from the supported modes. Either we can do it as a part of this patch or in a separate one.

Reply:
will do this in another PR